addon/components/tooltip-on-element.js

```diff
@@ -67,8 +67,11 @@ export default EmberTetherComponent.extend({
   'aria-hidden': computed.not('tooltipIsVisible'),
   attributeBindings: ['aria-hidden', 'role', 'tabindex'],
   classNameBindings: ['effectClass'],
-  classPrefix: 'tooltip',
-  classNames: ['tooltip'],
+  classPrefix: 'ember-tooltip',
+  classNames: ['ember-tooltip', 'tooltip'],
```

Comment:
I think we should decide if the two classes solution is overkill or not and how this will be released. I'll defer to @sir-dunxalot that was only my suggestion.... 
That said, if we decide to go with 2 classes, I think we need to also apply two classes for postitionClass, effectClass and typeClass for the same reasons.... So given the complexity it may introduce, I'm leaning more towards not doing it. I think this deserves a major change, not strictly an "API change", but certainly a breaking change.